server/plugin/api.go

```diff
@@ -1196,10 +1198,29 @@ func (p *Plugin) getMilestones(c *UserContext, w http.ResponseWriter, r *http.Re
 	p.writeJSON(w, allMilestones)
 }
 
-func getRepositoryList(c context.Context, userName string, githubClient *github.Client, opt github.ListOptions) ([]*github.Repository, error) {
+func getOrganizationList(c context.Context, userName string, githubClient *github.Client, opt github.ListOptions) ([]*github.Organization, error) {
+	var allOrgs []*github.Organization
```

Comment:
This API endpoint needs to take into account the `GitHubOrg` plugin config setting. If this setting is set, we should just return that one org in the setting's value.

https://github.com/mattermost/mattermost-plugin-github/blob/a45515e2fed91f6a3c040e05c002c6666c1a4f7d/server/plugin/configuration.go#L28

Reply:
Done.